docs/decisions/0019-forced-redemption.rst

```diff
@@ -0,0 +1,52 @@
+0019 Forced Redemption
+**********************
+
+Status
+======
+
+Accepted - April 2024
+
+Context
+=======
+There is frequently a need to force through a redemption (and related enrollment/fulfillment)
+of a particular learner, covered by a particular subsidy access policy, into
+some specific course run. This needs exists for reasons related to upstream
+business constraints, notably in cases where a course is included in a policy's catalog,
+but the desired course *run* is not discoverable due to the current state of its metadata.
```

Comment:
nit: would it be helpful to include a mention of what specific metadata is of concern here (e.g., `is_enrollable`)?